packs/sensu/actions/silence.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: silence
+runner_type: run-local-script
+description: Silence a Sensu client or check
+enabled: true
+entry_point: silence.py
+parameters:
+  check:
+    type: string
+    description: The check to silence
```

Comment:
client is required, check is optional.  That is why I made client a required parameter ;)